diff --git a/Assets/MRTK/Shaders/MixedRealityStandard.shader b/Assets/MRTK/Shaders/MixedRealityStandard.shader
index 93bc2e0..06f2cc4 100644
--- a/Assets/MRTK/Shaders/MixedRealityStandard.shader
+++ b/Assets/MRTK/Shaders/MixedRealityStandard.shader
@@ -772,7 +772,7 @@ Shader "Mixed Reality Toolkit/Standard"
                 return o;
             }
 
-            fixed4 frag(v2f i, fixed facing : VFACE) : SV_Target
+            fixed4 frag(v2f i) : SV_Target
             {
                 UNITY_SETUP_INSTANCE_ID(i);
 
@@ -953,10 +953,10 @@ Shader "Mixed Reality Toolkit/Standard"
                 worldNormal.x = dot(i.tangentX, tangentNormal);
                 worldNormal.y = dot(i.tangentY, tangentNormal);
                 worldNormal.z = dot(i.tangentZ, tangentNormal);
-                worldNormal = normalize(worldNormal) * facing;
+                worldNormal = normalize(worldNormal);
 #endif
 #else
-                worldNormal = normalize(i.worldNormal) * facing;
+                worldNormal = normalize(i.worldNormal);
 #endif
 #endif
 
